docs/source/copydbinprod.rst

```diff
@@ -0,0 +1,137 @@
+.. _copydbinprod:
+
+===========================================
+Running ``ghostferry-copydb`` in Production
+===========================================
+
+Assuming you have gone through :ref:`tutorialcopydb`, you probably want to run
+``ghostferry-copydb`` in production. The general workflow is relatively
+similar, with some differences. You should keep the tutorial as a starting
+point for your own playbook as most steps will largely be the same.
+
+Prerequisites
+-------------
+
+Before you start, you need to know if you can even use Ghostferry. Some points
+to consider about this are:
+
+- Ghostferry on its own does not enable 0 downtime moves. The downtime for the
+  app will be minimal compared to other methods but still non-zero.
+
+  - Figure out how much downtime you are willing to tolerate. Using Ghostferry,
+    one can realistically achieve downtime on the order of seconds.
+
+- The source database must be running with `FULL image`_ `ROW based replication`_.
+
+  - Without this, it is not possible to run Ghostferry safely.
```

Comment:
Is it possible to run it at all? If yes, it should probably just error (instead of running not-safely)?

Reply:
Yeah we have a check in the beginning of the run that will abort it if binlog_row_image is not FULL. I'll add it into the docs.